dali/pipeline/executor/workspace_policy.h

```diff
@@ -376,6 +391,18 @@ template <>
 struct AOT_WS_Policy<SeparateQueuePolicy> {
   AOT_WS_Policy() : depths_(0) {}
 
+  ~AOT_WS_Policy() {
+    for (auto &q : cpu_workspaces_)
+      for (auto &ws : q)
+        SetOrder(ws, AccessOrder::host());
```

Comment:
What is it for? To sync?

Reply:
Yes; probably no longer necessary, since we use CUDAStreamPool

Reply:
Can you remove to make code cleaner?